x-pack/solutions/search/plugins/serverless_search/public/plugin.ts

```diff
@@ -151,6 +152,15 @@ export class ServerlessSearchPlugin
       visibleIn: [],
       async mount({}: AppMountParameters) {
         const [coreStart] = await core.getStartServices();
+        const query = new URLSearchParams(window.location.search);
+        const onboardingToken = query.get('onboarding_token');
+        // note: test with http://localhost:5601/app/cloud/onboarding?next=/app/elasticsearch&onboarding_token=vector
```

Comment:
was this comment added intentionally and would it be removed before this PR is merged?

Reply:
Yes intentional, no it won't be removed--it's useful to have a clear way of testing this in the code.